File: js/linearSearch.js
exports.linearSearch = function(valueToFind, arrayToSearchThrough) {
    let answer;

    for (let i = 0; i < arrayToSearchThrough.length; i++) {
        valueToFind == arrayToSearchThrough[i] 
        ? answer = arrayToSearchThrough.indexOf(valueToFind) 
        : answer
    }
    
    return answer
};

exports.globalLinearSeach = function(valueToFind, arrayToSearchThrough) {
    let answer = []

    for (let i = 0; i < arrayToSearchThrough.length; i++) {
        valueToFind === arrayToSearchThrough[i] 
        ? answer.push(i)
        : answer
    }
    return answer
};

File: python/linear_search.py
def linear_search(value_to_find, array_to_search_through):
    answer = []

    for i, element in enumerate(array_to_search_through):
        # print(value_to_find, element, value_to_find == element)
        if value_to_find == element:
            answer.append(i)
    
    return answer if len(answer) else None

